lib/trento/hosts/host.ex

```diff
@@ -129,6 +129,7 @@ defmodule Trento.Hosts.Host do
     field :socket_count, :integer
     field :os_version, :string
     field :provider, Ecto.Enum, values: Provider.values()
+    field :prometheus_targets, :map, default: %{}
```

Comment:
I think we could omit the default here, like we don't put defaults in commands/events this could be easier when we want in the future spot what instances has the targets

Reply:
yes, I did initially this because otherwise as the default value is `nil`, the json view conversion was failing. But I can add a condition there to return empty targets list if the `prometheus_targets` value is nil.
I will add a patch

Reply:
@CDimonaco Change done.
I have removed the defaults. After doing this, I have done other tiny change, to still leave the "current" way of creating targets, just in case older agents are sending data, so we still have the current behaviour.
Removing will be super easy at any point, when we decide.